packages/app/components/jsonschemaform/jsonschemaform.tsx

```diff
@@ -85,8 +94,38 @@ const JsonSchemaForm = ({
             }}
             noValidate={allowValidationErrors}
             noHtml5Validate={allowValidationErrors}
+            validate={validate}
         />
     )
 }
 
 export default JsonSchemaForm
+
+//* Validate custom widgets here because of this issue: https://github.com/rjsf-team/react-jsonschema-form/issues/2718
+function validateCustomWidgets(formData: any, errors: FormValidation) {
```

Comment:
I really don't love this solution, but this is the best I can come up with until RJSF [allows custom widgets to raise errors](https://github.com/rjsf-team/react-jsonschema-form/issues/2718).

I don't like that `formData.templates` is dynamically defined but a static dependency of this code. Someone could change the schema and this validation would silently fail (it just wouldn't validate).